tachyon/crypto/commitments/fri/two_adic_fri.h

```diff
@@ -337,17 +337,17 @@ class TwoAdicFRI {
       absl::Span<const Eigen::Map<const math::RowMajorMatrix<F>>> matrices =
           matrices_by_round[round];
       const OpeningPointsForRound& points = points_by_round[round];
-      for (const Eigen::Map<const math::RowMajorMatrix<F>>& matrix : matrices) {
+      for (size_t i = 0; i < matrices.size(); ++i) {
+        const Eigen::Map<const math::RowMajorMatrix<F>>& matrix = matrices[i];
+        const std::vector<ExtF>& point_list = points[i];
```

Comment:
I think we have typically named vectors as `xxxs` and double vectors as `xxxs_list`.
So how about changing the variable names like `points` -> `points_list` and `points_list` -> `points`?

Reply:
I agree that the current naming choice is not ideal. Though `points` -> `points_vec` and `points_list` -> `points` is a good choice, I think naming `points` -> `points_for_round` and `points_list` -> `points_for_mat` would be better as it's more descriptive and easier for the reader to understand the logic. What do you think?

Reply:
I agree with adding more information to the naming. However, just from the name `points` or `points_for_round`, it might be difficult to tell that they are a double vector.

Reply:
Ah that's true. What about `points` -> `points_vec_for_round` and `points_list` -> `points_for_mat`?